aws-py-voting-app/blogpost.md

```diff
@@ -60,9 +61,9 @@ app_routetable_association = aws.ec2.MainRouteTableAssociation("app_routetable_a
     vpc_id=app_vpc)
 ```
 
-
-To inform what kinds of internet traffic are and aren't allowed to connect with the application, 
+To control internet traffic that are and aren't allowed to connect with the application,
 we create a firewall in the form of a security group.
```

Comment:
it's not really a firewall, so let's go with

"to control traffic that is allowed to flow between applications running inside our VPC, we create a security group"